src/vm/arm/asmhelpers.S

```diff
@@ -129,7 +129,8 @@ LOCAL_LABEL(LReturnDone):
         vldm    sp, {d0-d3}
 #endif
 
-        EPILOG_STACK_RESTORE_OFFSET   r7, #8
+        sub     r7, r7, #8
```

Comment:
My thought was that the macro will then have the implicit side effect of modifying the `Register` argument. Which is not good to my opinion.